app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -114,21 +118,78 @@ fun PocketStory(
 }
 
 /**
- * Displays a list of [PocketRecommendedStory]es on 3 by 3 grid.
+ * Displays a single [PocketSponsoredStory].
+ *
+ * @param story The [PocketSponsoredStory] to be displayed.
+ * @param onStoryClick Callback for when the user taps on this story.
+ */
+@Composable
+fun PocketSponsoredStory(
+    story: PocketSponsoredStory,
+    onStoryClick: (PocketSponsoredStory) -> Unit
+) {
+    val (imageWidth, imageHeight) = with(LocalDensity.current) {
+        116.dp.toPx().roundToInt() to 84.dp.toPx().roundToInt()
+    }
+    val imageUrl = story.imageUrl.replace(
+        "&resize=w[0-9]+-h[0-9]+".toRegex(),
+        "&resize=w$imageWidth-h$imageHeight"
+    )
+
+    ListItemTabSurface(imageUrl, { onStoryClick(story) }) {
+        PrimaryText(
+            text = story.title,
+            fontSize = 14.sp,
+            overflow = TextOverflow.Ellipsis,
+            maxLines = 2,
+        )
+
+        Spacer(Modifier.height(9.dp))
+
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Icon(
+                painter = painterResource(id = R.drawable.pocket_star_stroke),
+                contentDescription = null,
+                tint = FirefoxTheme.colors.iconSecondary,
```

Comment:
This and the text color below are slightly different than on Figma to not have to add a custom color.
Asked Sam from UX for validating any one approach.

Reply:
Apparently the slightly different color was just a design quirk.
Okayed to use the same `textSecondary` / `iconSecondary` colors.